### PR TITLE
{math}[foss/2018a] Theano v1.0.4

### PR DESCRIPTION
**Pull request by henkela**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/Theano/Theano-1.0.4-foss-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+description = """Theano is a Python library that allows you to define, optimize,
+and evaluate mathematical expressions involving multi-dimensional arrays efficiently."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['35c9bbef56b61ffa299265a42a4e8f8cb5a07b2997dabaef0f8830b397086913']
+dependencies = [('Python', '3.6.4')]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/t/Theano/Theano-1.0.4-foss-2018a-Python-3.6.4.eb]: Please add
```
use_pip = True
download_dep_fail = True
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2570.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6897c9cbd79c7d72a2f50954c22653ef for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2112.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9302f5fa06c024f2a9ca4a64cd59ea6c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar04.gastly.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/620620126c3b5d6295cd183ae7691661 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @henkela!

**Pull request merged**